lib/libutee/tee_api_operations.c

```diff
@@ -329,35 +330,52 @@ TEE_Result TEE_SetOperationKey(TEE_OperationHandle operation,
 	if (key != TEE_HANDLE_NULL) {
 		TEE_ObjectInfo key_info;
 
-		TEE_GetObjectInfo(key, &key_info);
+		res = TEE_GetObjectInfo1(key, &key_info);
+		if (res != TEE_SUCCESS && res != TEE_ERROR_CORRUPT_OBJECT &&
+		    res != TEE_ERROR_STORAGE_NOT_AVAILABLE)
+			TEE_Panic(0);
+
+		if (res == TEE_ERROR_CORRUPT_OBJECT ||
```

Comment:
If this test and action is done before the panic test at line 334 above, then the panic test can be simplified.